### ISSUE TITLE
Create New Year's themed banner

### ISSUE THREAD

**Issue by kwzrd** [python-discord/branding#128]
The [New Year event](https://github.com/python-discord/branding/tree/master/events/new_year) currently lacks a banner asset.

For a simple solution, the contribution can simply adapt the [Evergreen banner](https://github.com/python-discord/branding/blob/master/seasonal/evergreen/banner.png), use the [New Year icon](https://github.com/python-discord/branding/blob/master/events/new_year/annual.png), and apply its colours.

More ambitious contributions could incorporate usual motives of New Year's celebration. :confetti_ball: :fireworks: :wine_glass: 

Feel free to discuss ideas here or in `#media-branding` on Discord.

**Comment by dawnofmidnight** [comment]
I can work on this. I've done the base, I just need to add the accents.

**Comment by kwzrd** [comment]
Hi @dawnofmidnight, how is the banner coming along?

**Comment by dawnofmidnight** [comment]
Almost done with it, I'm just finishing up adding some effects in the background because the plan with the Os was not working very well.

**Comment by gustavwilliam** [comment]
I assume this is completed and can be closed now?

**Comment by kwzrd** [comment]
As 179292f64ae64dd93c144c6808fbaf4bed0f8973 explains, the version added in #140 is not considered complete and will be updated in the future. We decided to merge it in the current state to unblock #129. However, writing a separate issue to finish the work may make more sense than leaving this one open.

**Comment by dawnofmidnight** [comment]
I agree with kwzrd, this should probably be closed.

**Comment by gustavwilliam** [comment]
Since we have a banner now, I'm closing this. If someone's interested in creating a new one at a future date, like you were talking about, that'll be a different issue.
